python/triton/runtime/driver.py

```diff
@@ -63,6 +63,7 @@ def __init__(self):
         self.cuMemAlloc = mod.cuMemAlloc
         self.cuMemcpyHtoD = mod.cuMemcpyHtoD
         self.cuMemFree = mod.cuMemFree
+        self.cu_occupancy_max_active_clusters = mod.cu_occupancy_max_active_clusters
```

Comment:
would you mind renaming `cu_occupancy_max_active_clusters` to `cuOccupancyMaxActiveClusters`?

Reply:
Actually  there are two naming styles for this those variables, e.g. get_device_properties. I chose the orinal one and it may more align with exsiting codes when we use this function in test files, does it make sense?